src/IonUnicode.ts

```diff
@@ -27,82 +27,82 @@ const LOW_SURROGATE_OFFSET = 0xDC00;
 const LOW_SURROGATE_END = 0xE000;
 
 function isHighSurrogate(charCode: number) : boolean {
-  return charCode >= HIGH_SURROGATE_OFFSET && charCode < LOW_SURROGATE_OFFSET;
+    return charCode >= HIGH_SURROGATE_OFFSET && charCode < LOW_SURROGATE_OFFSET;
 }
 
 export function encodeUtf8(s: string) : number[] {
-  let writeable: Writeable = new Writeable(s.length, s.length);
-  for (let b of encodeUtf8Stream(charCodes(s))) {
-    writeable.writeByte(b);
-  }
-  return writeable.getBytes();
+    let writeable: Writeable = new Writeable();
+    for (let b of encodeUtf8Stream(charCodes(s))) {
+        writeable.writeByte(b);
+    }
+    return writeable.getBytes();
 }
 
-export function *encodeUtf8Stream(it: IterableIterator<number>) : IterableIterator<number> {
-  let pushback: number = -1;
-  for (;;) {
-    let codePoint: number;
-    if (pushback !== -1) {
-      codePoint = pushback;
-      pushback = -1;
-    } else {
-      let r: IteratorResult<number> = it.next();
-      if (r.done) {
-        return;
-      } else {
-        codePoint = r.value;
-      }
-    }
+export function *encodeUtf8Stream(it: IterableIterator<number>)  : IterableIterator<number> {
+    let pushback: number = -1;
+    for(;;) {
+        let codePoint: number;
+        if (pushback !== -1) {
+            codePoint = pushback;
+            pushback = -1;
+        } else {
+            let r: IteratorResult<number> = it.next();
+            if (r.done) {
+                return;
+            } else {
+                codePoint = r.value;
+            }
```

Comment:
not sure because I cannot tell from the browser but are we adding tabs here or spaces? We should be consistent.

Reply:
I've been trying not to do a ton of indent manipulation to avoid polluting the pr.